spec/Helpers.coffee

```diff
@@ -1517,22 +1635,23 @@ describe 'Component traits', ->
                 y.disconnect()
             , delay*req.num
 
-      noflo.helpers.WirePattern c,
-        in: ['x', 'y']
-        out: 'out'
-        async: true
-        forwardGroups: true
-        group: isUuid
-        gcFrequency: 2 # every 2 requests
-        gcTimeout: 0.02 # older than 20ms
-      , (input, groups, out, done) ->
-        setTimeout ->
-          out.send
-            id: groups[0]
-            x: input.x
-            y: input.y
-          done()
-        , 3
+      before ->
+        noflo.helpers.WirePattern c,
+          in: ['x', 'y']
+          out: 'out'
+          async: true
+          forwardGroups: true
+          group: isUuid
+          gcFrequency: 2 # every 2 requests
```

Comment:
grouping by RegExp is supported (top-level group only, see [here](https://github.com/noflo/noflo/pull/526/commits/68c7c52992de1dcf7cda57b41939f69a9d60b675#diff-2d415ed643629dc01235928209dcffc9R464). `gc` is indeed unnecessary here.